pyi.py

```diff
@@ -146,6 +152,10 @@ def run_check(self, plugin, **kwargs):
 class PyiVisitor(ast.NodeVisitor):
     filename: Path = Path("(none)")
     errors: list[Error] = field(default_factory=list)
+    # Mapping of all private TypeVars/ParamSpecs/TypeVarTuples to the nodes where they're defined
+    typevarlike_defs: dict[TypeVarInfo, ast.Assign] = field(default_factory=dict)
+    # Mapping of each name in the file to the no. of occurences
+    all_name_occurences: Counter[str] = field(default_factory=Counter)
```

Comment:
```suggestion
    all_name_occurrences: Counter[str] = field(default_factory=Counter)
```
This needs to be changed in a few other places

Reply:
Ahh good catch